src/Data/Functor/Clown.purs

```diff
@@ -0,0 +1,28 @@
+module Data.Functor.Clown where
+
+import Prelude
+
+import Data.Functor.FunctorRight (class FunctorRight)
+import Data.Newtype (class Newtype)
+
+-- | Make a `Functor` over the first argument of a `Bifunctor`
```

Comment:
With the reference to `` `Bifunctor` ``, this doesn't seem quite appropriate anymore. Although honestly I can't say the wording makes a lot of sense to me in its original context either.

What would people think about changing the doc comment to something like this?
```purs
-- | Lift a one-parameter type constructor to a two-parameter type constructor
-- with a phantom second parameter
```

Reply:
How about
> Makes a type that takes two type parameters (e.g. `Tuple`) into a `Functor` by operating on the first parameter rather than the second:
>
> ```purescript
> -- Functor instance for normal Tuple
> map show (Tuple 1 2) == (Tuple 1 "2")
> 
> -- Functor instance for Clowned Tuple
> map show (Clown (Tuple 1 2)) == Clown (Tuple "1" 2)

Reply:
I don't think that's [what a Clown does](https://try.purescript.org/?gist=fb312845f43c781f7bfbb864780b1b02), though? `map`ping over a Clown has no effect on the inner value, only the phantom argument. And I don't think there needs to be special attention paid to the case when Clown is applied to a two-parameter type versus a one-parameter type, does there? A `Clown (Tuple 1 2.0)` has type `Clown (Tuple Int) Number _`, and the `Int` part doesn't have any effect on further clownery, right? (It's been a while since I've read about clowns and jokers so I could be missing something important.)

Reply:
Oh, whoops. You're right.

Reply:
How about this then?
> Lifts a type that takes one type parameter to a type that takes two type parameters where the second parameter is a phantom type.
>
> ```purescript
> data Box a = Box a
> -- these values are the same at runtime
> Box a
> Clown (Box a) :: Clown a Int
> Clown (Box a) :: Clown a String
>
> type TupleInt x = Tuple Int x
> -- these values are the same at runtime
> Tuple 4 "foo" :: TupleInt String
> Clown (Tuple 4 "foo") :: Clown (TupleInt String Boolean)
> Clown (Tuple 4 "foo") :: Clown (TupleInt String Number)
> ```

Reply:
I like the general direction, although none of that actually type-checks. (The first set of examples need to be `Clown Box a Int`, etc.; the second set needs to have the parentheses around `TupleInt ...` removed and `TupleInt` itself needs to be defined without the `x` parameter so that it isn't partially applied, or it needs to be a `newtype` or something.)